placideirandora/epic-mail
# #164149028 built functional password reset content

#### What does this PR do?
It adds password reset functionality to the sign in content password reset link.

#### Description of Task to be completed?
1. Added a password reset modal to the password reset link with javascript.
2. When the password reset link is clicked on, the modal gets open with modal content.
3. The modal content is email field for entering an email for receiving a password reset link and a reset button for clicking on when having entered the email.
4. When the reset button is clicked on, a notification message pops up indicating that the password reset link has been sent to the email.

#### How should this be manually tested?
1. Navigate to the sign in page and click on the password reset link.
2. Then, Check if the above/mentioned procedures function accordingly.
